plugins/scoreboard/derma/cl_scoreboard.lua

```diff
@@ -98,7 +98,7 @@ local PANEL = {}
 				local faction = nut.faction.indices[k]
 				
 				if (amount != 0) then
-					v:SetVisible(hook.Run("HideFactionInScoreboard",faction) != false or LocalPlayer():IsAdmin())
+					v:SetVisible(hook.Run("HideFactionInScoreboard", faction) or LocalPlayer():IsAdmin())
```

Comment:
`hook.Run("HideFactionInScoreboard", faction)` by itself would return nil, which is not what we want as that would cause nothing to draw on the scoreboard unless the player is an admin, we want to specifically check if false was returned or not.